Simplified/Account.swift

```diff
@@ -321,10 +325,12 @@ private let accountSyncEnabledKey        = "NYPLAccountSyncEnabledKey"
 
 
   /// Load authentication documents from the network or cache.
```

Comment:
I would add here that this might also trigger the presentation of an announcement if one is present.